### PR TITLE
Fix/validate config

### PR DESCRIPTION
**Pull request by matthdsm**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-cmgg/preprocessing/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -249,3 +241,12 @@ manifest {
...
 includeConfig 'conf/modules.config'
+
+validation {
+    // Schema validation default options
+    monochromeLogs = false
+    lenientMode    = false
+    failUnrecognisedParams = false
+    showHiddenParams = false
+    defaultIgnoreParams = 'genomes,igenomes_base'
+}
```
Comment (nvnieuwk) [inline nextflow.config:252]:
```suggestion
    help {
        enabled: true
        command: "<fill in your command here>"
    }
}
```

File: nextflow.config [hunk 2]
```diff
@@ -59,7 +59,7 @@ params {
     // Schema validation default options
     validationFailUnrecognisedParams = false
     validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base'
+    validationSchemaIgnoreParams     = ""
     validationShowHiddenParams       = false
```
Comment (nvnieuwk) [inline nextflow.config:63]: These can be removed! only `validate_params` should stay ;)

Comment (nvnieuwk) [reply nextflow.config:63]: Unless you want to keep the parameters of course

Comment (matthdsm) [reply nextflow.config:63]: the params are used to configurate the plugin
https://github.com/nf-cmgg/preprocessing/pull/97/files#diff-ce7465a8e67b3b9c95696db1146ca7e6328f075e08a3d64062b87aa8608fc4eaR252-R265

Comment (nvnieuwk) [reply nextflow.config:63]: Yes I know but what is the benefit of doing this?

**Pull request closed without merging**